cluster/etcd_service_discovery_test.go

```diff
@@ -493,7 +493,8 @@ func TestParallelGetter(t *testing.T) {
 
 	parallelGetter := newParallelGetter(cli, 5)
 	for _, serverToAdd := range serversToAdd {
-		parallelGetter.addWork(serverToAdd.Type, serverToAdd.ID)
+		payload := []byte("{\"id\":\"" + serverToAdd.ID + "\",\"type\":\"" + serverToAdd.Type + "\",\"frontend\":true,\"hostname\":\"" + serverToAdd.ID + "\",\"metadata\":{\"region\":\"us\"}}")
```

Comment:
we shouldn't assume this payload, if I add a new parameter to the metadata, this will fail, right? if so then this is a no-go

Reply:
You can add more params to metadata and it won't fail, the code is just not checking for any additional parameters at this time. This dummy payload is just to ensure code coverage for the same existing functions

Reply:
ooooh ok, I didn't see this was a test file!